DigitalLearningSolutions.Web/Views/TrackingSystem/CourseSetup/AdminFields/_NoConfiguredAnswers.cshtml

```diff
@@ -0,0 +1,5 @@
+﻿<p class="nhsuk-body-m">
+  There are no answers configured for this registration prompt.
```

Comment:
This looks like it has been copied but the text not updated to be relevant to the current page. Either we need to use the same partial for both this and the registration prompts if we want the same text, or the text here needs updating. (I suspect the text here needs updating)

Reply:
Ah good catch, just an oversight on my part. I've changed it to say "There are no answers configured for this admin field."